crashlytics/Shared/UITests.swift

```diff
@@ -0,0 +1,72 @@
+//
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import XCTest
+
+class UITests: XCTestCase {
+  override func setUpWithError() throws {
+    // Put setup code here. This method is called before the invocation of each test method in the class.
+
+    // In UI tests it is usually best to stop immediately when a failure occurs.
+    continueAfterFailure = false
+
+    // In UI tests it’s important to set the initial state - such as interface orientation - required for your tests before they run. The setUp method is a good place to do this.
+  }
+
+  override func tearDownWithError() throws {
+    // Put teardown code here. This method is called after the invocation of each test method in the class.
+  }
+
+  func testStaticUI() throws {
+    // UI tests must launch the application that they test.
+    let app = XCUIApplication()
+    app.launch()
+
+    // Use recording to get started writing UI tests.
+    // Use XCTAssert and related functions to verify your tests produce the correct results.
+
+    #if !os(macOS)
+      XCTAssertTrue(
+        app.navigationBars["Crashlytics Example"].exists,
+        "Crashlytics Example is missing from the navigation bar"
+      )
```

Comment:
Assertions should be written affirming the point you expect to be true, because if the test fails they'll show up in the logs as
```
expected "Crashlytics Example exists in the navigation bar" to be true
```

Reply:
The second string of XCTAssertTrue is a description of the failure. https://developer.apple.com/documentation/xctest/xctasserttrue

Reply:
Whoops, thanks for the correction :)